### ISSUE TITLE
Auth Service dies on bad requests

### ISSUE THREAD

**Issue by corysabol** [ProfessionallyEvil/arrrspace#9]
## Error reproduction
- Create cluster with `./setup.sh` or apply the latest configs `kubectl apply -f k8s-resources/`
- `curl -XPOST 172.17.0.2:31337/auth/legacy/register` or any endpoint it seems
- view status with `kubectl get pods --watch` in another terminal

This only seems to happen when supplying malformed or missing body params, etc. I have not enumerated all of the failure cases.

A fix would be to have the auth service gracefully response to malformed or incorrect inputs.

**Comment by mgillam** [comment]
Yeah, authSvc needs a bunch more work probably. It's currently cobbled together just to have a means of issuing JWTs.
